tools/harness-thci/OpenThread_BR.py

```diff
@@ -612,9 +612,12 @@ def mdns_query(self, dst='ff02::fb', service='_meshcop._udp.local', addrs_blackl
         print('mdns_query %s %s %s' % (dst, service, addrs_blacklist))
 
         result = self.bash('dig -p 5353 @%s %s ptr +time=2 +retry=2' % (dst, service))
-        responses = ' '.join(result).split(';; ANSWER SECTION:')
+
+        if dst in 'ff02::fb' and not addrs_blacklist:
```

Comment:
```suggestion
        if dst == 'ff02::fb' and not addrs_blacklist:
```

Looks like `==` is better.

Reply:
Done.